.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,94 @@
+name: Bug Report
+description: Create a bug report
+labels: ["type: bug"]
+body:
+
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for contributing with Quarkus OpenAPI Generator with this bug report! Submit your issue below:
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Bug Report
+
+  - type: dropdown
+    id: extension
+    attributes:
+      label: "Tell us the extension you're using"
+      multiple: false
+      options:
+        - Client
+        - Server
+        - Mock
+    validations:
+      required: true
+
+  - type: textarea
+    id: i-tried-this
+    attributes:
+      label: "I tried this:"
+      placeholder: "What did you try to do? A code snippet or example helps."
+    validations:
+      required: true
+
+  - type: textarea
+    id: instead-what-happened
+    attributes:
+      label: "This happened:"
+      placeholder: "What happened instead of what you've expected?"
+    validations:
+      required: true
+
+  - type: textarea
+    id: what-did-you-expect
+    attributes:
+      label: "I expected this:"
+      placeholder: "What did you expect to happen? Describe the output or behavior you expected to see (unless it's obvious)."
+
+  - type: textarea
+    id: workaround
+    attributes:
+      label: "Is there a workaround?"
+      placeholder: "What's the workaround to avoid this issue?"
+
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: "How can we try to reproduce the issue?"
+      placeholder: "What steps or configuration do we need to reproduce the erratic behaviour?"
+
+  - type: textarea
+    attributes:
+      label: Anything else?
+      placeholder: |
+        Links? References? Logs? Anything that will give us more context about the issue you are encountering.
+        Tip: You can attach images or log files by dragging files in.
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Environment
```

Comment:
I think that the `Environment` do not expose meaning, we can add more options here:

```yaml
- type: input
    attributes:
      label: Output of `uname -a` or `ver`
  - type: input
    attributes:
      label:  Output of `java -version`
  - type: input
    attributes:
      label:  Quarkus OpenApi version or git rev
  - type: input
    attributes:
      label:  Build tool (ie. output of `mvnw --version` or `gradlew --version`)
  - type: textarea
    attributes:
      label: Additional information
      description: >
        If you have any additional information for us, use the field below.
        Please note, you can attach screenshots or screen recordings here, by
        dragging and dropping files in the field below.
```